frontend/src/common/NotFoundIcon.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import posed, { PoseGroup } from 'react-pose';
+
+const AnimatablePath = posed.path({
```

Comment:
Are this kind of animations agreed with client and design? And are they prioritized over the functionality we are missing?